src/components/CourtOrderPoa/CourtOrderPoa.stories.ts

```diff
@@ -0,0 +1,51 @@
+import { CourtOrderPoa } from './index'
+
+export default {
+  title: 'component/CourtOrderPoa',
+  component: CourtOrderPoa,
+  argTypes: {
+  }
+}
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { CourtOrderPoa },
+  template: '<court-order-poa v-bind="$props" />' // $props comes from args below
+})
+
+// const ContactData: ContactPointIF = {
```

Comment:
Obsolete code to delete?

Also below.

Reply:
OOPS, lots of clean up required here.

Reply:
The story was much simpler than i expected lol